create-record/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const
+    AWS = require('aws-sdk'),
+    uuid = require('uuid'),
+    S3 = new AWS.S3();
+
+exports.handler = async (event, context) => {
+
+    const bucket = process.env.dataBucket;
+    const filePath = process.env.fileLoca;
+    const kmsKey = process.env.dataKey;
+    let filename = uuid.v1();
+    let transactionStatus = "FAILED";
+    var body = JSON.stringify(event);
```

Comment:
This should be a const as it's never modified and we don't need to hoist this variable.

Reply:
I can make filename a const no problem. I usually reserve const to define what should never be changed as a note to future developers, In this case we don't change file name but it is not restricted to not be changed later if a naming convention is chosen. I will flip it to const.